Tests/SwiftCurrent_SwiftUITests/SwiftCurrent_SwiftUITests.swift

```diff
@@ -228,6 +238,36 @@ final class SwiftCurrent_SwiftUIConsumerTests: XCTestCase {
         wait(for: [expectViewLoaded], timeout: 0.3)
     }
 
+    func testWorkflowOnlyShowsOneViewAtATime() throws {
```

Comment:
I'm not sure this test name matches the asserts.  If you showed all the views at the same time, wouldn't the test down below still pass?

Reply:
Nope, the trick is at the bottom. ViewInspector doesn't support `find(FR1.self, skipFound 1)` so I used `ViewType.Text.self`. This is in relation to #76 where I talked about how if you don't have an `if` statement for casting the body when you have the same `FlowRepresentable` twice in a `Workflow` it presents both of them. This test proves that does *not* happen.